File: fix.py
import os

files = {
    'src/app/(store)/checkout/page.tsx': [
        ("\% : \$}", "\% : $\}"),
        ("\", "\"),
        ("\", "\"),
        ("\", "\")
    ],
    'src/app/(store)/invoice/[id]/page.tsx': [
        ("{w-8 h-8  + ", "{\"w-8 h-8 \" + ")
    ],
    'src/app/(store)/product/[slug]/page.tsx': [
        ("{w-5 h-5  + ", "{\"w-5 h-5 \" + "),
        ("{w-4 h-4  + ", "{\"w-4 h-4 \" + "),
        ("\", "\")
    ],
    'src/app/api/invoices/[id]/simulate-payment/route.ts': [
        ("allDeliveredContent.push(Product:  + product.name + \\n + deliveredContent);",
         "allDeliveredContent.push(\"Product: \" + product.name + \"\\n\" + deliveredContent);")
    ]
}

for path, replacements in files.items():
    with open(path, 'r', encoding='utf-8') as f:
        content = f.read()
    for old, new in replacements:
        content = content.replace(old, new)
    with open(path, 'w', encoding='utf-8') as f:
        f.write(content)

File: fix.js
const fs = require('fs');

let c1 = fs.readFileSync('src/app/(store)/checkout/page.tsx', 'utf8');
c1 = c1.replace(/\$\{discount\.value\}% : \\$/g, '\$%\ : $\');
c1 = c1.replace(/\\\$\/g, '$$'); // wait, let's just do simple string replacements
fs.writeFileSync('src/app/(store)/checkout/page.tsx', c1);

let i1 = fs.readFileSync('src/app/(store)/invoice/[id]/page.tsx', 'utf8');
i1 = i1.replace(/className=\{w-8 h-8  \+ /g, 'className={"w-8 h-8 " + ');
fs.writeFileSync('src/app/(store)/invoice/[id]/page.tsx', i1);

let p1 = fs.readFileSync('src/app/(store)/product/[slug]/page.tsx', 'utf8');
p1 = p1.replace(/className=\{w-5 h-5  \+ /g, 'className={"w-5 h-5 " + ');
p1 = p1.replace(/className=\{w-4 h-4  \+ /g, 'className={"w-4 h-4 " + ');
fs.writeFileSync('src/app/(store)/product/[slug]/page.tsx', p1);

let s1 = fs.readFileSync('src/app/api/invoices/[id]/simulate-payment/route.ts', 'utf8');
s1 = s1.replace(/allDeliveredContent\.push\(Product:  \+ /g, 'allDeliveredContent.push("Product: " + ');
s1 = s1.replace(/\+ \\n \+ /g, '+ "\\n" + ');
fs.writeFileSync('src/app/api/invoices/[id]/simulate-payment/route.ts', s1);

